src/style.css

```diff
@@ -1,3 +1,77 @@
-body {
+:root {
+  --active: #00ACDC;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+}
+
+nav {
+  margin-right: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+nav ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+nav ul a {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  margin-left: 20px;
+  height: 60px;
+}
+
+nav ul a:hover {
+  color: var(--active);
+}
+
+a.is-active {
+  display: flex;
+  position: relative;
+  color: var(--active);
+}
+
+a.is-active::after {
+  display: flex;
+  position: absolute;
+  border-radius: 8px;
+  content: '';
+  border: 2px solid var(--active);
+  width: 100%;
+  box-sizing: border-box;
+  align-self: flex-end;
+  justify-content: center;
+}
+
+.nav ul li:first-child a {
```

Comment:
1. You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children

2. you can use `:not(:first-child)` to set margin for all `li` besides first one